components/chainhook-cli/src/scan/bitcoin.rs

```diff
@@ -219,7 +219,7 @@ pub async fn execute_predicates_action<'a>(
                 actions_triggered += 1;
                 match action {
                     BitcoinChainhookOccurrence::Http(request) => {
-                        send_request(request, 3, 1, &ctx).await?
+                        send_request(request, 10, 3, &ctx).await
                     }
                     BitcoinChainhookOccurrence::File(path, bytes) => {
                         file_append(path, bytes, &ctx)?
```

Comment:
The above (line 222) removed a `?`. Was that on purpose? If so, this line needs it removed as well, there is a mismatched return type on this match.

Reply:
Thank you for catching these snafus!
Addressed with latest.